CONTRIBUTING_DOCS.md

```diff
@@ -158,24 +151,51 @@ Here are some other shortcodes:
 - `readfile`: Include the content of another file in the current file, which can be in an arbitrary location.
 - `bootstrap-table`: formats a table using Bootstrap classes; accepts any bootstrap table classes as additional arguments, e.g. `{{< bootstrap-table "table-bordered table-hover" }}`
 
+### Add code to documentation pages
+
+For command, binary and process names, you can use a pair of backticks (\`\<some-name\>\`): `<some-name>`.
```

Comment:
```suggestion
For command, binary and process names, you can use a pair of backticks (\`\<some-name\>\`): `<some-name>`.
```

Reply:
@travisamartin Was there meant to be change here?

Reply:
Oops -- I mean to add the serial comma. Fixed now.